app/design/adminhtml/Magento/backend/web/css/source/forms/_temp.less

```diff
@@ -462,6 +462,18 @@ label.mage-error {
     }
 }
 
+.admin__fieldset{
+    &.downloadable{
```

Comment:
Please, move Downloadable product related styles to the corresponded module instead of Backend